generators/gulp/templates/gulpfile.js

```diff
@@ -66,7 +58,7 @@ gulp.task('test', ['pre-test'], function (cb) {
 
 gulp.task('coveralls', ['test'], function () {
   if (!process.env.CI) {
-    return;
+    return false;
```

Comment:
Not sure but it seems like http://eslint.org/docs/rules/consistent-return - I just want `return;` to be valid in cases like this.